app/src/main/java/com/antosik/benchproject/app/more/view/MoreFragment.kt

```diff
@@ -3,16 +3,23 @@ package com.antosik.benchproject.app.more.view
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.ViewGroup
-import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.antosik.benchproject.app.common.navigator.Navigator
+import com.antosik.benchproject.app.more.viewModel.MoreViewModel
 import com.antosik.benchproject.databinding.MoreFragmentBinding
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
-class MoreFragment : Fragment() {
+class MoreFragment : Navigator<MoreFragmentNavigator>() {
```

Comment:
The base class for fragments with navigation should be rather named `NavigableFragment` or something like that, `Navigator` should be for navigator classes not for fragment/activity.

Reply:
Ok, I understand, so I changed it 😄